src/content-handlers/iiif/extensions/uv-pdf-extension/config/config.json

```diff
@@ -153,10 +153,12 @@
         "expandFull": "$expandGallery",
         "holdingText": "$moduleGoesHere",
         "less": "$less",
+        "lessAriaLabelPrefix": "Less information: Hide",
```

Comment:
You should use `$lessAriaLabelPrefix` instead of the raw string in all of these config.json files. Same for `$moreAriaLabelPrefix` below. Otherwise I don't think the i18n will work.